src/jennifer/relation/many_to_many.cr

```diff
@@ -72,6 +74,14 @@ module Jennifer
         @association_foreign || Inflector.foreign_key(T.to_s)
       end
 
+      def association_primary_key
+        if (association_primary = @association_primary)
```

Comment:
what about 
```crystal
association_primary ? T.c(association_primary) : T.primary
# or
return T.c(association_primary) if association_primary

T.primary

```

Reply:
I definitely like the first one. I will have to go see about the `T#c` method for future reference. Ill update this branch today or tomorrow.